demo/index.js

```diff
@@ -20,6 +20,11 @@ init({
 const logEvents = ['error', 'load', 'volumeChanged', 'captionsChanged', 'playerStatusChanged', 'videoLoaded', 'seeked',
 	'webcastLoaded', 'webcastStarted', 'webcastEnded', 'broadcastStarted', 'broadcastStopped'];
 
+document.querySelector('#updateToken').addEventListener('click', () => {
+	const data = getData();
+	currentEmbed?.updateToken(data.config.token);
```

Comment:
this should be vbrickEmbed, not currentEmbed

Reply:
The variable's name is currentEmbed.